fv3core/utils/gt4py_utils.py

```diff
@@ -52,8 +52,61 @@
 # Logger instance
 logger = logging.getLogger("fv3ser")
 
+
+def get_size() -> int:
+    """Get the size of the MPI COMM_WORLD communicator if it is defined.
+
+    Returns:
+        int: size
+    """
+    return MPI.COMM_WORLD.Get_size() if MPI else 0
```

Comment:
I understand why this was put here (so that it can be used in `decorators.py`), but this seems like the wrong place for MPI utilities...

Reply:
what is your suggestion?

Reply:
If they're used within decorators, can they just be defined there? Also, probably want them to begin with a leading `_`.